pkg/index/index.go

```diff
@@ -0,0 +1,82 @@
+package index
+
+import (
+	"io/ioutil"
+	"net/http"
+	"sort"
+	"time"
+
+	"github.com/blang/semver/v4"
+	"gopkg.in/yaml.v2"
+	"helm.sh/helm/v3/pkg/chart"
+)
+
+const (
+	defaultIndexURL = "https://raw.githubusercontent.com/GreptimeTeam/helm-charts/gh-pages/index.yaml"
+)
+
+type ChartVersions []*ChartVersion
```

Comment:
Can we just import `ChartVersion` and `IndexFile` from the Helm library? If can't, use `third_party/` to import external code.